packages/ssr/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './createBrowserClient';
+export * from './createServerClient';
+export * from './types';
+
+export {
```

Comment:
@silentworks It looks like all we are excluding here is the `isSecureEnvironment` function (see other comment). If we removed that function, we could just `export * from './utils';`

Reply:
Have removed function and exported all utils